tests/src/Kernel/CrossMenuAssignmentTest.php

```diff
@@ -38,11 +44,8 @@ protected function setUp(): void {
     $this->installEntitySchema('localgov_menu_link_group');
     $this->container->get('plugin.manager.menu.link')->rebuild();
 
-    $admin_user = User::create([
-      'name' => 'admin',
-      'mail' => 'admin@example.net',
-    ]);
-    $admin_user->addRole('administrator');
+    $admin_user = $this->createUser([], 'admin', TRUE, ['mail' => 'admin@example.net']);
```

Comment:
Is EntityKernelTestBase just here for this?

Interested rather than bothered.

Reply:
Yes and I believe some other things it does differently, trying the other testbase didn't seem to work with assigning roles correctly (at least in Drupal 10.3) until I did some digging around other contrib modules that seem to make similar changes